core-web/libs/ui/src/lib/dot-tab-buttons/dot-tab-buttons.component.ts

```diff
@@ -27,12 +27,11 @@ export class DotTabButtonsComponent {
     @Output() clickOption = new EventEmitter();
```

Comment:
Adding the click to a div have a11y issues, why don't put the click directly to the buttons?

Reply:
Also... if the parent div should be the host, this: https://dotcms.slack.com/archives/C01VC00SA5A/p1694626261079379